### PR TITLE
feat: change the order of the nav

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

___
### New feature or improvement
- Describe the details and related test reports.
change the order of the nav

### CODE REVIEW

File: web/src/helpers.tsx [hunk 1]
```diff
@@ -61,11 +61,6 @@ export const getMenuData = (): MenuDataItem[] => {
       path: '/ssl/list',
       icon: <IconFont name="iconssl" />,
     },
-    {
```
Comment (juzhiyuan) [inline web/src/helpers.tsx:64]: Where?

Comment (LiteSun) [reply web/src/helpers.tsx:64]: why remove setting?

**Review by LiteSun — CHANGES_REQUESTED** [review]

File: web/src/helpers.tsx (hunk 1 above)
Comment (juzhiyuan) [reply web/src/helpers.tsx:64]: image

The sidebar has too many tools, we could put Settings in Navbar.

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 2]
```diff
@@ -17,10 +17,23 @@
 /* eslint-disable no-undef */
 
 context('settings page smoke test', () => {
-  const domSelectors = {
+  const data = {
+    grafana_address: 'Grafana Address',
+    grafana_explain1: 'Grafana address should begin with HTTP or HTTPS',
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:22]: `should start with xx`

Comment (juzhiyuan) [reply web/cypress/integration/settings/settings-smoketest.spec.js:22]: `explanation`

File: web/src/helpers.tsx (hunk 1 above)
Comment (LiteSun) [reply web/src/helpers.tsx:64]: Got it.

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 3]
```diff
@@ -17,10 +17,23 @@
...
+    grafana_explain2: 'Address is illegality',
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:23]: `Invalid URL`

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 4]
```diff
@@ -17,10 +17,23 @@
...
+    update_success: 'Update Configuration Successfully',
+    url1: 'httx://www.test.com',
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:25]: Use `invalidURL` would be more reasonable.

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 5]
```diff
@@ -17,10 +17,23 @@
 /* eslint-disable no-undef */
 
 context('settings page smoke test', () => {
-  const domSelectors = {
+  const data = {
+    grafana_address: 'Grafana Address',
+    grafana_explain1: 'Grafana address should begin with HTTP or HTTPS',
+    grafana_explain2: 'Address is illegality',
+    update_success: 'Update Configuration Successfully',
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:24]: `Update Successfully`

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 6]
```diff
@@ -18,20 +18,20 @@
...
-    update_successfully: 'Update Configuration Successfully',
-    invalid_URL: 'httx://www.test.com',
-    new_URL: 'https://apisix.apache.org/',
-    fetch_URL: 'fetchURL',
+    grafanaAddress: 'Grafana Address',
+    grafanaExplain1: 'Grafana address should begin with HTTP or HTTPS',
+    grafanaExplain2: 'Address is illegality',
+    updateSuccessfully: 'Update Configuration Successfully',
+    invalidURL: 'httx://www.test.com',
+    ValidURL: 'https://apisix.apache.org/',
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js]:
```suggestion
    validURL: 'https://apisix.apache.org/',
```

File: web/cypress/integration/settings/settings-smoketest.spec.js [hunk 7]
```diff
@@ -18,20 +18,20 @@
...
-    grafana_address: 'Grafana Address',
-    start_explanation: 'Grafana address should begin with HTTP or HTTPS',
-    after_explanation: 'Address is illegality',
-    update_successfully: 'Update Configuration Successfully',
-    invalid_URL: 'httx://www.test.com',
-    new_URL: 'https://apisix.apache.org/',
-    fetch_URL: 'fetchURL',
+    grafanaAddress: 'Grafana Address',
+    grafanaExplain1: 'Grafana address should begin with HTTP or HTTPS',
+    grafanaExplain2: 'Address is illegality',
```
Comment (juzhiyuan) [inline web/cypress/integration/settings/settings-smoketest.spec.js:23]: `Explain` is a variable, use `explanation`.

File: web/cypress/integration/route/create-route-with-upstream.spec.js [hunk 8]
```diff
@@ -88,6 +89,7 @@ context('Create Route with Upstream', () => {
     cy.visit('/');
     cy.contains('Route').click();
 
+    cy.get(domSelector.refresh).click();
```
Comment (LiteSun) [inline web/cypress/integration/route/create-route-with-upstream.spec.js:92]: why need to click refresh here? what about using `cy.reload()`?🤔

**Review by juzhiyuan — APPROVED** [review]

File: web/src/locales/en-US/menu.ts [hunk 9]
```diff
@@ -71,5 +71,5 @@ export default {
   'menu.plugin': 'Plugin',
   'menu.service': 'Service',
   'menu.setting': 'Settings',
-  'menu.serverinfo': 'Server Info',
```
Comment (juzhiyuan) [inline web/src/locales/en-US/menu.ts:74]: Should use `menu.systemStatus` actually

**Review by LiteSun — APPROVED** [review]

**Comment by imjoey** [comment]
@guoqqqi Thank you very much for the work. While could you please provide more details about why to do this and what this PR will do? An elaborated description would help both reviewers and other developers to well understand the PR. Much appreciated. 😄

**Comment by juzhiyuan** [comment]
> @guoqqqi Thank you very much for the work. While could you please provide more details about why to do this and what this PR will do? An elaborated description would help both reviewers and other developers to well understand the PR. Much appreciated. 😄

@guoqqqi Good suggestion, a clear description with some snapshots would better.

**Pull request merged**